---
title: Taming the BEAST Down Under
layout: workshop
address: Charles Perkins Centre, University of Sydney
location: Sydney, Australia
latitude: -33.8873
longitude: 151.1835
startdate: 2019-02-17
enddate: 2019-02-22
image: /images/workshops/Taming-the-BEAST-Down-Under-Logo.png
homepage: https://sydney.edu.au/science/biology/meep/workshops/
flyer: /images/workshops/Taming-the-BEAST-Down-Under-Flyer.jpg

has_programme: true
---

# Workshop Programme

The following is a <b>preliminary</b> programme and will be subject to change.

<table>
<thead>

<tr>
<th></th>
<th> Sunday <br> 2019/02/17 </th>
<th> Monday <br> 2019/02/18</th>
<th> Tuesday <br> 2019/02/19 </th>
<th> Wednesday <br> 2019/02/20 </th>
<th> Thursday <br> 2019/02/21 </th>
<th> Friday <br> 2019/02/22 </th>
</tr>

</thead>

<tbody>

<tr>
<td> 09:00</td>
<td></td>
<td> <b>Alexei Drummond</b> <br> Bayesian Phylogenetics and BEAST </td>
<td> <b>Remco Bouckaert</b> <br> Phylogenetic model selection and averaging </td>
<td> <b>TBA</b> <br> Birth-death Bayesian Phylodynamics: Skylines, Structure and Fossilization </td>
<td> <b>Mike Lee</b> <br> Bayesian phylogenetics for phenotypic and palaeontological datasets </td>
<td> <b>Sebastián Duchêne</b> <br> Tip-dating and phylodynamics </td>
</tr>

<tr>
<td> 11:00 </td>
<td></td>
<td> Morning Tea </td>
<td> Morning Tea </td>
<td> Morning Tea </td>
<td> Morning Tea </td>
<td> Morning Tea </td>
</tr>


<tr>
<td> 11:30 </td>
<td></td>
<td rowspan="2"> <b>Tim Vaughan</b>
    <br> First Steps in BEAST&nbsp;2<br>
    (lecture and tutorial) </td>
<td rowspan="2"> <b>Remco Bouckaert</b> <br> Substitution model averaging <br>
    (tutorial) </td>
<td> <b>TBA</b> <br> Public Lecture </td>
<td rowspan="2"> <b>Jana Huisman</b> <br> Fossilized birth-death model analyses <br> (tutorial) </td>
<td rowspan="2"> <b>Nicola Müller</b> <br> Isolation with Migration using Coupled MCMC <br> (tutorial) </td>
</tr>

<tr>
<td> 12:30 </td>
<td></td>
<td rowspan="9" style="border: solid 1px gray"> Free afternoon (packed lunch provided)</td>
</tr>

<tr>
<td> 13:00 </td>
<td></td>
<td>Lunch</td>
<td>Lunch</td>
<td>Lunch</td>
<td>Lunch</td>
</tr>

<tr>
<td> 14:00 </td>
<td></td>
<td rowspan="2"> <b>Jana Huisman</b> <br> Prior Selection &amp; Troubleshooting <br> (lecture and tutorial)</td>
<td rowspan="2"> <b>Simon Ho </b> <br> Molecular Dating</td>
<td> <b>David Duchêne</b> <br> Species trees and gene trees</td>
<td rowspan="2"> <b>Tim Vaughan</b> <br> Bacterial genomics </td>
</tr>

<tr>
<td> 15:00 </td>
<td></td>
<td> Afternoon Tea </td>
</tr>


<tr>
<td> 15:30 </td>
<td></td>
<td> Afternoon Tea </td>
<td> Afternoon Tea </td>
<td rowspan="3"> <b>Jérémie Sciré &amp; Nicola Müller</b> <br>
    Analyzing structured populations under birth-death and coalescent models <br>
    (tutorial)
</td>
<td> Afternoon Tea </td>
</tr>

<tr>
<td> 16:00 </td>
<td rowspan="4" style="border: solid 1px gray">
Welcome and registration<br> <i>University of Sydney</i></td>
<td rowspan="2"> <b>Nicola Müller</b> <br> Skyline Plot Tutorial </td>
<td rowspan="2"> <b>Cara Van Der Wal &amp; Perry Beasley-Hall</b> <br> Divergence time estimation</td>
<td rowspan="4" style="border: solid 1px gray">
Departure</td>
</tr>

<tr>
<td> 17:00 </td>
</tr>

<tr>
<td> 18:00 </td>
<td> </td>
<td> </td>
<td rowspan="2">
Buffet dinner cruise<br>
(Sydney Harbour, Depart from King St Wharf)</td>
</tr>

<tr>
<td> 18:30 </td>
<td> Dinner (Newtown)</td>
<td> </td>
</tr>

</tbody>
</table>

{% if page.repo %}

# Lecture slides

Slides to lectures are available from [here]( https://github.com/{{ page.repo }} )

{% endif %}


# Required Software

**You are required to bring your own laptop to the workshop.** Power strips are available for each participant to keep their devices charged during the workshop. The software below **must** be installed prior to arrival. 

- **Java 8 or higher:** This is required by BEAST 2 [(download)](http://java.com/download).
- **BEAST 2:** BEAST 2 is a cross-platform program for Bayesian phylogenetic analysis of molecular sequences. We will be using this software to perform analyses on the data brought to the workshop. We will be using **BEAST v2.5.1** for the workshop [(download)](http://beast2.org/).
- **FigTree:** FigTree is a graphical viewer of phylogenetic trees, specifically designed to display summarised and annotated trees produced by BEAST [(download)](http://beast.community/figtree).
- **Tracer:** Tracer is a program for analysing the trace files generated by Bayesian MCMC runs. It can be used to check mixing, ESS, parameter correlations, etc. [(download)](http://beast.community/tracer).
- **BEAGLE (optional):** BEAGLE is a high-performance library that can perform the core calculations in BEAST2 (and many other phylogenetics packages) much faster. BEAGLE isn't necessary for the workshop, but it will speed up your analyses [(download)](https://github.com/beagle-dev/beagle-lib).
- **R:** R is a free software environment that provides a wide variety of statistical (linear and nonlinear modelling, classical statistical tests, time-series analysis, classification, clustering) and graphical techniques, and is highly extensible [(download)](https://www.r-project.org/).
- **RStudio (optional):** RStudio provides a friendly front-end for R [(download)](https://www.rstudio.com/).


# Pre-workshop recommended reading


### Statistics and mathematics primers

We are aware that some of the workshop participants do not have a mathematical background and may have difficulty understanding some of the theory. The two links below can help to refresh the basic concepts, but are by no means meant to be comprehensive resources.

- [Seeing Theory](http://students.brown.edu/seeing-theory/index.html): This website gives a very brief, but pretty and easy to understand introduction to many of the fundamental concepts in statistics. 
- Ziheng Yang's [statistics primer](http://abacus.gene.ucl.ac.uk/PPS/PrimerProbabilityStatistics.pdf): Gives a more in-depth introduction to the topic.

### Phylogenetics and phylodynamics books

The books below are not all focused on BEAST or even Bayesian phylogenetics. However, each of the books contains at least one background chapter on the theory of molecular evolution, as well as more in-depth chapters on the models implemented in BEAST and BEAST 2. Most of the books also contain at least some discussion of Bayesian inference.

- [Bayesian Evolutionary Analysis with BEAST](https://www.beast2.org/book/), _Alexei J. Drummond, Remco R. Bouckaert_
- [Molecular Evolution: A Statistical Approach](http://abacus.gene.ucl.ac.uk/MESA/), _Ziheng Yang_
- [The Phylogenetic Handbook](http://www.cambridge.org/catalogue/catalogue.asp?isbn=9780521877107), _Philippe Lemey, Marco Salemi, Anne-Mieke Vandamme_
- [Molecular Evolution: A Phylogenetic Approach](http://eu.wiley.com/WileyCDA/WileyTitle/productCd-0865428891.html), _Roderick D.M. Page, Edward C. Holmes_
- [Inferring Phylogenies](https://www.amazon.co.uk/Inferring-Phylogenies-Joseph-Felsenstein/dp/0878931775), _Joseph Felsenstein_

In addition, the following review papers might be useful:

- [Bayesian molecular clock dating of species divergences in the genomics era](https://www.nature.com/articles/nrg.2015.8), _Mario dos Reis, Phillip Donoghue, Ziheng Yang_
- [Evolutionary analysis of the dynamics of viral infectious disease](https://www.nature.com/articles/nrg2583), _Oliver Pybus, Andrew Rambaut_
- [Bayesian molecular dating: opening up the black box](https://onlinelibrary.wiley.com/doi/abs/10.1111/brv.12390), _Lindell Bromham, Sebastián Duchêne, Xia Hua, Andrew M. Ritchie, David A. Duchêne, Simon Y. W. Ho_
