---
title: Taming the BEAST Eh!
layout: workshop
location: Squamish, British Columbia
startdate: 2019-08-12
enddate: 2019-08-16
image: /images/workshops/Taming-the-BEAST-Squamish-Logo.png
homepage: https://ttbcanada.ca/

flyer: /images/workshops/Taming-the-BEAST-Squamish-Flyer.jpg

has_programme: true
---

# Workshop Programme

<table>
<thead>

<tr>
<th></th>
<th> Sunday <br> 2018/08/11 </th>
<th> Monday <br> 2018/08/12</th>
<th> Tuesday <br> 2018/08/13 </th>
<th> Wednesday <br> 2018/08/14 </th>
<th> Thursday <br> 2018/08/15 </th>
<th> Friday <br> 2018/08/16 </th>
</tr>

</thead>

<tbody>

<tr>
<td> 07:00 </td>
<td rowspan="14"> Arrival </td>
<td class="food"> Breakfast and Late Registration </td>
<td class="food"> Breakfast </td>
<td class="food"> Breakfast </td>
<td class="food"> Breakfast </td>
<td class="food"> Breakfast </td>
</tr>

<tr>
<td> 08:45 </td>
<td> Welcome Message<br>
<b>Jeffrey Joy</b> <br>
BC Centre for Excellence in HIV/AIDS </td>
<td> Housekeeping </td>
<td> Housekeeping </td>
<td> Housekeeping </td>
<td> Housekeeping </td>
</tr>

<tr>
<td> 09:00 </td>
<td class="intvited-lecture"> Invited Lecture <br>
Macroevolution, phylogenetics and rate time dependency<br>
<b>Matthew Pennell</b><br>
University of British Columbia</td>
<td rowspan="2" class="lecture"> Lecture <br>
Tree Priors, Clocks and Skylines <br>
<b> Alexei Drummond </b> </td>
<td class="lecture"> Lecture <br>
Statistical Methods to Reconstruct Phylogenetic Networks<br>
<b> Claudia Solís-Lemus</b> </td>
<td class="lecture"> Lecture <br>
Divergence Time Dating and the Fossilized Birth-Death Model <br>
<b> Joëlle Barido-Sottani</b> </td>
<td class="lecture"> Lecture <br>
Species Trees with StarBeast2 <br>
<b> Huw Ogilvie </b> </td>
</tr>

<tr>
<td> 10:00 </td>
<td class="lecture"> Lecture <br>
Basic Principles of Statistical Inference <br>
<b> Alexei Drummond </b> </td>
<!-- <td> </td> -->
<td class="tutorial"> Tutorial 5 <br>
Bayesian Skyline Plot and Birth-Death Skyline Tutorial <br>
<b> Louis du Plessis </b> </td>
<td class="tutorial"> Tutorial 6 <br>
Fossilized Birth-Death Model <br>
<b> Joëlle Barido-Sottani </b> </td>
<td> Tutorial 9 <br>
Calibrating Species Trees <br>
<b> Huw Ogilvie </b> </td>
</tr>

<tr>
<td> 11:00 </td>
<td rowspan="2" class="food"> Morning Tea &amp; Coffee </td>
<td rowspan="2" class="food"> Morning Tea &amp; Coffee </td>
<td class="food"> Morning Tea &amp; Coffee </td>
<td rowspan="2" class="food"> Morning Tea &amp; Coffee </td>
<td rowspan="2" class="food"> Morning Tea &amp; Coffee </td>
</tr>

<tr>
<td> 11:20 </td>
<td rowspan="2" class="invited-lecture">
Invited Lecture <br>
Evolutionary History of HBV in the Circumpolar Arctic <br>
<b> Carla Oslowy <br>
National Microbiology Laboratory <br>
/ University of Manitoba </b>
</td>
</tr>

<tr>
<td> 11:30 </td>
<td rowspan="2" class="lecture"> Lecture <br>
First Steps in BEAST 2 <br>
<b> Louis du Plessis </b> </td>
<td rowspan="2" class="lecture"> Lecture <br>
Model Selection and Averaging <br>
<b> Remco Bouckaert </b> </td>
<td rowspan="2" class="lecture"> Lecture <br>
Structured Coalescent Approximations <br>
<b> David Rasmussen </b> </td>
<td rowspan="2" class="invited-lecture"> Invited Lecture <br>
Phylogenetic inferences for Big Data <br>
<b> Claudia Solís-Lemus <br>
Emory University </b> </td>
</tr>

<tr>
<td> 12:30 </td>
<td rowspan="8"> Free Afternoon for Exploring <br>
(Bagged Lunch Provided) </td>
</tr>

<tr>
<td> 13:00 </td>
<td class="food"> Lunch </td>
<td class="food"> Lunch </td>
<td class="food"> Lunch <br>
<b> Quest University </b> </td>
<td class="food"> Lunch <br>
<b> Quest University </b> </td>
</tr>

<tr>
<td> 14:00 </td>
<td class="tutorial"> Tutorial 1 <br>
Prior Selection and Clock Calibration <br>
<b> Louis du plessis </b> </td>
<td class="tutorial"> Tutorial 3 <br>
Model Averaging <br>
<b> Remco Bouckaert </b> </td>
<td class="tutorial"> Tutorial 7 <br>
Phylogeography in BEAST 2 <br>
<b> Remco Bouckaert </b> </td>
<td> Troubleshooting Personal Projects and Closing Remarks </td>
</tr>

<tr>
<td> 15:00 </td>
<td class="food"> Afternoon Tea &amp; Coffee </td>
<td class="food"> Afternoon Tea &amp; Coffee </td>
<td class="food"> Afternoon Tea &amp; Coffee </td>
<td class="food"> Afternoon Tea &amp; Coffee </td>
</tr>

<tr>
<td> 15:30 </td>
<td class="lecture"> Lecture <br>
Introduction to Infectious Disease Phylodynamics <br>
<b> David Rasmussen </b> </td>
<td class="lecture"> Lecture <br>
Troubleshooting and Summarizing BEAST 2 Output <br>
<b> Jūlija Pečerska </b> </td>
<td class="lecture"> Lecture <br>
Advanced Birth-Death Models <br>
<b> Jūlija Pečerska </b> </td>
<td rowspan="4"> Departure </td>
</tr>

<tr>
<td> 16:30 </td>
<td class="tutorial"> Tutorial 2 <br>
Relaxed Phylogenetics in BEAST 2 <br>
<b> Christiaan Swanepoel </b> </td>
<td class="tutorial"> Tutorial 4 <br>
Troubleshooting BEAST 2 <br>
<b> Jūlija Pečerska </b> </td>
<td class="tutorial"> Tutorial 8 <br>
Population Structure using the Multitype Birth-Death Model <br>
<b> Jūlija Pečerska </b> </td>
</tr>

<tr>
<td> 17:30 </td>
<td></td>
<td></td>
<td></td>
</tr>

<tr>
<td> 19:00 </td>
<td class="food"> Welcome &amp; Registration <br>
<b> A-Frame Brewery Co. &amp; Luz Taco Truck </b> </td>
<td class="food"> Dinner </td>
<td class="food"> Dinner </td>
<td class="food"> Dinner <br>
<b> Howe Sound Brewery </b> </td>
</tr>

</tbody>
</table>

{% if page.repo %}

# Lecture slides

Slides to lectures are available from [here]( https://github.com/{{ page.repo }} )

{% endif %}


# Required Software

**You are required to bring your own laptop to the workshop.** Power strips are available for each participant to keep their devices charged during the workshop. The software below **must** be installed prior to arrival. 

- **Java 8 or higher:** This is required by BEAST 2 [(download)](http://java.com/download).
- **BEAST 2:** BEAST 2 is a cross-platform program for Bayesian phylogenetic analysis of molecular sequences. We will be using this software to perform analyses on the data brought to the workshop. We will be using **BEAST v2.6.0** for the workshop [(download)](http://beast2.org/).
- **FigTree:** FigTree is a graphical viewer of phylogenetic trees, specifically designed to display summarised and annotated trees produced by BEAST [(download)](http://beast.community/figtree).
- **Tracer:** Tracer is a program for analysing the trace files generated by Bayesian MCMC runs. It can be used to check mixing, ESS, parameter correlations, etc. [(download)](http://beast.community/tracer).
- **BEAGLE (optional):** BEAGLE is a high-performance library that can perform the core calculations in BEAST2 (and many other phylogenetics packages) much faster. BEAGLE isn't necessary for the workshop, but it will speed up your analyses [(download)](https://github.com/beagle-dev/beagle-lib).
- **R:** R is a free software environment that provides a wide variety of statistical (linear and nonlinear modelling, classical statistical tests, time-series analysis, classification, clustering) and graphical techniques, and is highly extensible [(download)](https://www.r-project.org/).
- **RStudio (optional):** RStudio provides a friendly front-end for R [(download)](https://www.rstudio.com/).


# Pre-workshop recommended reading


### Statistics and mathematics primers

We are aware that some of the workshop participants do not have a mathematical background and may have difficulty understanding some of the theory. The two links below can help to refresh the basic concepts, but are by no means meant to be comprehensive resources.

- [Seeing Theory](http://students.brown.edu/seeing-theory/index.html): This website gives a very brief, but pretty and easy to understand introduction to many of the fundamental concepts in statistics. 
- Ziheng Yang's [statistics primer](http://abacus.gene.ucl.ac.uk/PPS/PrimerProbabilityStatistics.pdf): Gives a more in-depth introduction to the topic.

### Phylogenetics and phylodynamics books

The books below are not all focused on BEAST or even Bayesian phylogenetics. However, each of the books contains at least one background chapter on the theory of molecular evolution, as well as more in-depth chapters on the models implemented in BEAST and BEAST 2. Most of the books also contain at least some discussion of Bayesian inference.

- [Bayesian Evolutionary Analysis with BEAST](https://www.beast2.org/book/), _Alexei J. Drummond, Remco R. Bouckaert_
- [Molecular Evolution: A Statistical Approach](http://abacus.gene.ucl.ac.uk/MESA/), _Ziheng Yang_
- [The Phylogenetic Handbook](http://www.cambridge.org/catalogue/catalogue.asp?isbn=9780521877107), _Philippe Lemey, Marco Salemi, Anne-Mieke Vandamme_
- [Molecular Evolution: A Phylogenetic Approach](http://eu.wiley.com/WileyCDA/WileyTitle/productCd-0865428891.html), _Roderick D.M. Page, Edward C. Holmes_
- [Inferring Phylogenies](https://www.amazon.co.uk/Inferring-Phylogenies-Joseph-Felsenstein/dp/0878931775), _Joseph Felsenstein_

In addition, the following review papers might be useful:

- [Bayesian molecular clock dating of species divergences in the genomics era](https://www.nature.com/articles/nrg.2015.8), _Mario dos Reis, Phillip Donoghue, Ziheng Yang_
- [Evolutionary analysis of the dynamics of viral infectious disease](https://www.nature.com/articles/nrg2583), _Oliver Pybus, Andrew Rambaut_
- [Bayesian molecular dating: opening up the black box](https://onlinelibrary.wiley.com/doi/abs/10.1111/brv.12390), _Lindell Bromham, Sebastián Duchêne, Xia Hua, Andrew M. Ritchie, David A. Duchêne, Simon Y. W. Ho_
