---
title: Squamish BEAST Tamers
layout: workshop
location: Squamish, British Columbia
startdate: 2023-08-14
enddate: 2023-08-18
image: /images/workshops/Taming-the-BEAST-SBT-Logo.png
homepage: https://ttbcanada.ca/
repo: Taming-the-BEAST/Squamish-BEAST-Tamers-2023-Lectures
flyer: /images/workshops/Taming-the-BEAST-SBT-Flyer.jpg

has_programme: true
---


# Workshop Programme

<table style="width:40%">
<tbody>
<tr>
<td class="invited-lecture"><b>Invited talks</b></td>
<td class="lecture"><b>Lectures</b></td>
<td class="tutorial"><b>Tutorials</b></td>
<td class="help-session"><b>Interactive</b></td>
</tr>
</tbody>
</table>


<table>
<thead>

<tr>
<th style="width:5%"></th>
<th style="width:5%">Sunday <br> 2023/08/13 </th>
<th style="width:18%">Monday <br> 2023/08/14</th>
<th style="width:18%">Tuesday <br> 2023/08/15 </th>
<th style="width:18%">Wednesday <br> 2023/08/16 </th>
<th style="width:18%">Thursday <br> 2023/08/17 </th>
<th style="width:18%">Friday <br> 2023/08/18 </th>
</tr>

</thead>

<tbody>

<!-- 8.45 -->
<tr>
<td> 08:45 </td>
<td rowspan="37" class="padding"> Arrival </td>
<td rowspan="2"> Welcome Message<br>
<b>Jeffrey Joy</b> <br>
(<a href="https://www.bccfe.ca/about-us/team/joy-j">BCCfE in HIV/AIDS</a>)</td>
<td> Housekeeping </td>
<td> Housekeeping </td>
<td> Housekeeping </td>
<td> Housekeeping </td>
</tr>

<!------------------------------------------------------------------>
<!-- 9.00-->
<tr>
<td> 09:00 </td>

<!-- Tue -->
<td rowspan="4" class="lecture"><b>Alexei Drummond</b><br>
								<i>Tree priors, clocks and skylines</i><br><!--(<a href="https://github.com/Taming-the-BEAST/Squamish-BEAST-Tamers-2023-Lectures/raw/master/">slides</a>)-->
</td>

<!-- Wed -->
<td rowspan="4" class="invited-lecture"><b>Ailene MacPherson</b><br>
										(<a href="https://www.sfu.ca/math/people/faculty/ailenem.html">Simon Fraser University</a>)
</td>

<!-- Thu -->
<td rowspan="2" class="invited-lecture"><b>Jūlija Pečerska</b><br>
										(<a href="https://www.zhaw.ch/en/about-us/person/pece/">Zürich University of Applied Sciences</a>)
</td>

<!-- Fri -->
<td rowspan="4" class="invited-lecture"><b>Sungsik (Kevin) Kong</b><br>
										(<a href="https://sungsik-kong.github.io">UW Madison</a>)</td>
</tr>

<!-- 9.15 -->
<tr>
<td> 09:15 </td>

<!-- Mon -->
<td rowspan="3" class="lecture"><b>Alexei Drummond</b><br>
								<i>Introduction to Bayesian phylogenetics</i><br><!--(<a href="https://github.com/Taming-the-BEAST/Squamish-BEAST-Tamers-2023-Lectures/raw/master/">slides</a>)-->
</td>
</tr>

<!-- 9.30 -->
<tr>
<td> 09:30 </td>
<!-- Thu -->
<td rowspan="2" class="invited-lecture"> <b>David Rasmussen</b><br>(<a href="https://phylodynamics.wordpress.ncsu.edu">NC State University</a>)
</td>
</tr>

<!-- 9.45 -->
<tr>
<td> 09:45 </td>
</tr>

<!------------------------------------------------------------------>
<!-- 10.00 -->
<tr>
<td> 10:00 </td>
<td class="food"><b> Break</b> </td>
<td class="food"><b> Break</b> </td>
<td class="food"><b> Break</b> </td>
<td class="food"><b> Break</b> </td>
<td class="food"><b> Break</b> </td>
</tr>

<!------------------------------------------------------------------>
<!-- 10:15 -->
<tr>
<td> 10:15 </td>

<!-- Mon -->
<td rowspan="3" class="lecture"><b>Louis du Plessis</b><br>
								<i>First steps in BEAST2</i><br><!--(<a href="https://github.com/Taming-the-BEAST/Squamish-BEAST-Tamers-2023-Lectures/raw/master/">slides</a>)-->
</td>

<!-- Tue -->
<td rowspan="3" class="invited-lecture"> <b>Michael Martin</b><br>(<a href="https://www.iddynamics.jhsph.edu/michael-martin">Johns Hopkins University</a>)</td>

<!-- Wed -->
<td rowspan="3" class="lecture"><b>Joëlle Barido-Sottani</b><br>
								<i>Divergence time estimation</i><br><!--(<a href="https://github.com/Taming-the-BEAST/Squamish-BEAST-Tamers-2023-Lectures/raw/master/">slides</a>)-->
</td>

<!-- Thu -->
<td rowspan="3" class="lecture"><b>Huw Ogilvie</b><br>
								<i>Species tree inference and the multispecies coalescent</i><br><!--(<a href="https://github.com/Taming-the-BEAST/Squamish-BEAST-Tamers-2023-Lectures/raw/master/">slides</a>)-->
</td>

<!-- Fri -->
<td rowspan="3" class="lecture"><b>Remco Bouckaert</b><br>
								<i>Model selection and model averaging</i><br><!--(<a href="https://github.com/Taming-the-BEAST/Squamish-BEAST-Tamers-2023-Lectures/raw/master/">slides</a>)-->
</td>
</tr>

<!-- 10:30 -->
<tr>
<td> 10:30 </td>
</tr>

<!-- 10:45 -->
<tr>
<td> 10:45 </td>
</tr>

<!------------------------------------------------------------------>
<!-- 11:00 -->
<tr>
<td> 11:00 </td>
<td rowspan="2" class="food"><b>Coffee Break</b></td>
<td rowspan="2" class="food"><b>Coffee Break</b></td>
<td rowspan="2" class="food"><b>Coffee Break</b></td>
<td rowspan="2" class="food"><b>Coffee Break</b></td>
<td rowspan="2" class="food"><b>Coffee Break</b></td>
</tr>

<!-- 11:15 -->
<tr>
<td> 11:15 </td>
</tr>


<!------------------------------------------------------------------>
<!-- 11:30 -->
<tr>
<td> 11:30 </td>

<!-- Mon -->
<td rowspan="4" class="tutorial"><b>Introduction to BEAST2</b><br>
								(<a href="/tutorials/Introduction-to-BEAST2/">tutorial</a>)</td>

<!-- Tue -->
<td rowspan="4" class="tutorial"><b>Inference of past population dynamics using skyline models</b><br>
								(<a href="/tutorials/Skyline-plots/">BEAST2 tutorial</a>
								,<a href="https://linguaphylo.github.io/tutorials/skyline-plots/">LPhyBEAST tutorial</a>)</td>

<!-- Wed -->
<td rowspan="4" class="tutorial"><b>Dating species divergences with the fossilized birth-death process</b><br>
								(<a href="/tutorials/FBD-tutorial/">tutorial</a>)</td>

<!-- Thu -->
<td rowspan="4" class="tutorial"><b>Estimating species trees using StarBEAST2</b><br>
								(<a href="/tutorials/starbeast2-tutorial/">tutorial</a>)</td>

<!-- Fri -->
<td rowspan="4" class="tutorial"><b>Substitution model averaging</b><br>
								(<a href="/tutorials/Substitution-model-averaging/">tutorial</a>)</td>

</tr>


<!-- 11:45 -->
<tr>
<td> 11:45 </td>
</tr>


<!-- 12:00 -->
<tr>
<td> 12:00 </td>
</tr>


<!-- 12:15 -->
<tr>
<td> 12:15 </td>
</tr>


<!------------------------------------------------------------------>
<!-- 12:30 -->
<tr>
<td> 12:30 </td>
<td rowspan="4" class="food"> <b>Lunch</b> <br><i>Howe Sound Brewery</i>  </td>
<td rowspan="4" class="food"> <b>Lunch</b> <br><i>Howe Sound Brewery</i>  </td>
<td rowspan="2" class="food"> <b>Collect packed lunch</b></td>
<td rowspan="4" class="food"> <b>Lunch</b> <br><i>Howe Sound Brewery</i>  </td>
<td rowspan="4" class="food"> <b>Lunch</b> <br><i>Howe Sound Brewery</i>  </td>
</tr>

<!-- 12:45 -->
<tr>
<td> 12:45 </td>
</tr>

<!-- 13:00 -->
<tr>
<td> 13:00 </td>
<td rowspan="20" class="padding"> Free afternoon</td>
</tr>

<!-- 13:15 -->
<tr>
<td> 13:15 </td>
</tr>

<!------------------------------------------------------------------>
<!-- 13:30 -->
<tr>
<td> 13:30 </td>

<td rowspan="4" class="invited-lecture"> <b>Caroline Colijn</b><br>
										(<a href="https://www.sfu.ca/math/people/faculty/ccolijn/">Simon Fraser University</a>)</td>
<td rowspan="2" class="lecture"><b>Jūlija Pečerska</b><br>
								<i>Troubleshooting in BEAST2</i><br><!--(<a href="https://github.com/Taming-the-BEAST/Squamish-BEAST-Tamers-2023-Lectures/raw/master/">slides</a>)--></td>

<td rowspan="4" class="lecture"><b>Joëlle Barido-Sottani</b> &amp;<br> <b>David Rasmussen</b><br>
								<i>Structured models in phylodynamics</i><br>
								<br><!--(<a href="https://github.com/Taming-the-BEAST/Squamish-BEAST-Tamers-2023-Lectures/raw/master/">slides</a>)--></td>
<td rowspan="4" class="help-session"><b>Free Q&amp;A session</b><br>(optional)</td>

</tr>

<!-- 13:45 -->
<tr>
<td> 13:45 </td>
</tr>

<!-- 14:00 -->
<tr>
<td> 14:00 </td>

<!-- Tue -->
<td rowspan="2" class="tutorial"><b>BEAST2 convergence</b><br>
								(<a href="/tutorials/Troubleshooting/">tutorial</a>)</td>

</tr>

<!-- 14:15 -->
<tr>
<td> 14:15 </td>
</tr>

<!------------------------------------------------------------------>
<!-- 14:30 -->
<tr>
<td> 14:30 </td>

<td rowspan="2" class="food"><b>Coffee Break</b> </td>
<td rowspan="2" class="tutorial"><b>BEAST2 initialization</b><br>
								(<a href="/tutorials/">tutorial</a>)</td>

<td rowspan="2" class="food"><b>Coffee Break</b> </td>
<td rowspan="1" class="padding">Departure</td>

</tr>

<!-- 14:45 -->
<tr>
<td> 14:45 </td>
</tr>


<!------------------------------------------------------------------>
<!-- 15:00 -->
<tr>
<td> 15:00 </td>

<td rowspan="4" class="tutorial"><b>Prior selection and clock calibration</b><br>
								(<a href="/tutorials/Prior-selection/">tutorial</a>)</td>
<td rowspan="2" class="food"><b>Coffee Break</b> </td>

<td rowspan="4" class="tutorial"><b>Parameter and state inference using the approximate structured coalescent</b><br>
								(<a href="/tutorials/Mascot-Tutorial/">BEAST2 tutorial</a>,
								,<a href="https://linguaphylo.github.io/tutorials/structured-coalescent/">LPhyBEAST tutorial</a>)</td>
</tr>

<!-- 15:15 -->
<tr>
<td> 15:15 </td>
</tr>

<!-- 15:30 -->
<tr>
<td> 15:30 </td>

<!-- Tue -->
<td rowspan="4" class="lecture"><b>Alexei Drummond</b><br>
								<i>LinguaPhylo</i><br><!--(<a href="https://github.com/Taming-the-BEAST/Squamish-BEAST-Tamers-2023-Lectures/raw/master/">slides</a>)--></td>

</tr>

<!-- 15:45 -->
<tr>
<td> 15:45 </td>
</tr>


<!------------------------------------------------------------------>
<!-- 16:00 -->
<tr>
<td> 16:00 </td>

<!-- Mon -->
<td rowspan="8" class="invited-lecture"><b>Participant introductions and flash talks</b><br></td>

<!-- Thu -->
<td rowspan="4" class="tutorial"><b>Population structure using the multi-type birth-death model</b><br>
								 (<a href="/tutorials/Structured-birth-death-model/">tutorial</a>)</td>

</tr>

<tr>
<td> 16:15 </td>
</tr>

<tr>
<td> 16:30 </td>

<!-- Tue -->
<td rowspan="4" class="tutorial"><b>Ancestral state reconstruction using discrete phylogeography</b><br>
								(<a href="https://linguaphylo.github.io/tutorials/discrete-phylogeography/">LPhyBEAST tutorial</a>)</td>

</tr>

<tr>
<td> 16:45 </td>
</tr>

<tr>
<td> 17:00 </td>

<!-- Thu -->
<td rowspan="4" class="help-session"><b>BEAST clinic</b><br>(optional)</td>

</tr>

<tr>
<td> 17:15 </td>
</tr>

<tr>
<td> 17:30 </td>

<!-- Tue -->
<td rowspan="2" class="help-session"><b>BEAST Clinic</b><br>(optional)</td>

</tr>

<tr>
<td> 17:45 </td>
</tr>


<!------------------------------------------------------------------>
<!-- 18:00 -->
<tr>
<td> 18:00 </td>

<td rowspan="1" class="food"> <b>Welcome event</b>  <br><i>A-Frame Brewing &amp; Flipside Burgers</i> </td>
<td rowspan="1" class="food"> <b>Dinner</b>  <br><i>Howe Sound Brewery</i> </td>
<td rowspan="1" class="food"> <b>Dinner</b>  <br><i>Howe Sound Brewery</i> </td>
<td rowspan="1" class="food"> <b>Dinner</b>  <br><i>Beach BBQ</i> </td>
<td rowspan="1" class="food"> <b>Farewell dinner</b>  <br><i>Free Bird</i> </td>

</tr>


</tbody>
</table>

# Instructors

- [Joëlle Barido-Sottani](https://bjoelle.github.io) (Institut de Biologie de l'ENS)
- [Remco Bouckaert](https://www.cs.auckland.ac.nz/~remco/) (University of Auckland)
- [Alexei Drummond](https://profiles.auckland.ac.nz/adru001) (University of Auckland)
- [Louis du Plessis](https://bsse.ethz.ch/cevo/the-group/people/person-detail.lduplessis.html) (ETH Zürich)
- [Huw Ogilvie](https://profiles.rice.edu/faculty/huw-ogilvie) (Rice University)
- [Alex Popinga](https://allisongroup.blogs.auckland.ac.nz/people/) (University of Auckland)
- [Jūlija Pečerska](https://www.zhaw.ch/en/about-us/person/pece/) (Zürich University of Applied Sciences)
- [David Rasmussen](https://phylodynamics.wordpress.ncsu.edu) (North Carolina State University)



{% if page.repo %}

# Lecture slides

Slides to lectures are available from [here]( https://github.com/{{ page.repo }} ).

{% endif %}


# Required Software

Please ensure the following software is installed prior to the start of the workshop.

- **Java 8 or higher:** This is required by BEAST 2 [(download)](https://adoptium.net/en-GB/).
- **BEAST 2:** BEAST 2 is a cross-platform program for Bayesian phylogenetic analysis of molecular sequences. We will be using this software to perform analyses on the data brought to the workshop. We will be using **BEAST v2.7** for the workshop [(download)](http://beast2.org/).
- **FigTree:** FigTree is a graphical viewer of phylogenetic trees, specifically designed to display summarised and annotated trees produced by BEAST [(download)](http://beast.community/figtree).
- **Tracer:** Tracer is a program for analysing the trace files generated by Bayesian MCMC runs. It can be used to check mixing, ESS, parameter correlations, etc. [(download)](http://beast.community/tracer).
- **BEAGLE (optional):** BEAGLE is a high-performance library that can perform the core calculations in BEAST2 (and many other phylogenetics packages) much faster. BEAGLE isn't necessary for the workshop, but it will speed up your analyses [(download)](http://beast.community/beagle).
- **R:** R is a free software environment that provides a wide variety of statistical (linear and nonlinear modelling, classical statistical tests, time-series analysis, classification, clustering) and graphical techniques, and is highly extensible [(download)](https://www.r-project.org/).
- **RStudio (optional):** RStudio provides a friendly front-end for R [(download)](https://posit.co/download/rstudio-desktop/).


# Pre-workshop recommended reading


### Statistics and mathematics primers

We are aware that some of the workshop participants do not have a mathematical background and may have difficulty understanding some of the theory. The two links below can help to refresh the basic concepts, but are by no means meant to be comprehensive resources.

- [Seeing Theory](http://students.brown.edu/seeing-theory/index.html): This website gives a very brief, but pretty and easy to understand introduction to many of the fundamental concepts in statistics. 
- Ziheng Yang's [statistics primer](http://abacus.gene.ucl.ac.uk/PPS/PrimerProbabilityStatistics.pdf): Gives a more in-depth introduction to the topic.

### Phylogenetics and phylodynamics books

The books below are not all focused on BEAST or even Bayesian phylogenetics. However, each of the books contains at least one background chapter on the theory of molecular evolution, as well as more in-depth chapters on the models implemented in BEAST and BEAST 2. Most of the books also contain at least some discussion of Bayesian inference.

- [Bayesian Evolutionary Analysis with BEAST](https://www.beast2.org/book/), _Alexei J. Drummond, Remco R. Bouckaert_
- [Molecular Evolution: A Statistical Approach](http://abacus.gene.ucl.ac.uk/MESA/), _Ziheng Yang_
- [The Phylogenetic Handbook](http://www.cambridge.org/catalogue/catalogue.asp?isbn=9780521877107), _Philippe Lemey, Marco Salemi, Anne-Mieke Vandamme_
- [Molecular Evolution: A Phylogenetic Approach](http://eu.wiley.com/WileyCDA/WileyTitle/productCd-0865428891.html), _Roderick D.M. Page, Edward C. Holmes_
- [Inferring Phylogenies](https://www.amazon.co.uk/Inferring-Phylogenies-Joseph-Felsenstein/dp/0878931775), _Joseph Felsenstein_

In addition, the following review papers might be useful:

- [Bayesian molecular clock dating of species divergences in the genomics era](https://www.nature.com/articles/nrg.2015.8), _Mario dos Reis, Phillip Donoghue, Ziheng Yang_
- [Evolutionary analysis of the dynamics of viral infectious disease](https://www.nature.com/articles/nrg2583), _Oliver Pybus, Andrew Rambaut_
- [Bayesian molecular dating: opening up the black box](https://onlinelibrary.wiley.com/doi/abs/10.1111/brv.12390), _Lindell Bromham, Sebastián Duchêne, Xia Hua, Andrew M. Ritchie, David A. Duchêne, Simon Y. W. Ho_
